atspi-common/src/macros.rs

```diff
@@ -353,3 +353,41 @@ macro_rules! event_test_cases {
 		assert_impl_all!(zbus::Message: TryFrom<$type>);
 	};
 }
+
+/// Asserts that the signatures are equal, but ignores the outer parentheses as
+/// the difference between marshalled and unmarshalled signatures is often just one set of outer parentheses.
+#[macro_export]
+macro_rules! assert_eq_signatures {
```

Comment:
I see why this is implemented separately from the plain function, but it's unfortunate that it has a completely separate code path. Any ideas here?

Reply:
The excuse is because I wrote the macro before I wrote the function and it never occurred to me to implement the macro in terms of the function. Its a good idea! Thanks!

Reply:
[x]   implement `assert_eq_signatues!` in terms of `signatures_are_eq()`.